README.md

```diff
@@ -2,6 +2,10 @@
 
 Drone plugin to build and publish Docker images to a container registry.
 
+### Special privilieges
+
+This docker image is specified in Drone to run with Privilieged mode. If you fork this repository to run it on your own, you need to specify your docker image in the `DRONE_ESCALATE`, as the container needs to run in privilieged mode, otherwise it will not work! https://github.com/drone/drone/blob/master/cmd/drone-server/server.go
```

Comment:
You got a typo in privileged multiple times, please drop the moving reference to the drone core source file, you should append "environment variable" to DRONE_ESCALATE.

Generally I'm not sure if the wording is that great.

Reply:
I can agree the wording can be better @tboerger :) How would it be said better?